libpod/image/docker_registry_options.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/containers/image/v5/docker/reference"
 	"github.com/containers/image/v5/types"
 
```

Comment:
Can you remove this blank line?

Reply:
I can if you'd like, but my understanding is that the blank line is intentional (it predates my patch), to separate in-package imports from external imports.

Reply:
Removed